plugins/analytics-module-ga/src/apis/implementations/AnalyticsApi/GoogleAnalytics.test.ts

```diff
@@ -152,6 +152,99 @@ describe('GoogleAnalytics', () => {
       });
     });
 
+    it('captures virtual pageviews', () => {
+      const config = new ConfigReader({
+        app: {
+          analytics: {
+            ga: {
+              trackingId,
+              testMode: true,
+              virtualSearchPageView: { mode: 'only' },
+            },
+          },
+        },
+      });
+      const api = GoogleAnalytics.fromConfig(config);
+      api.captureEvent({
+        action: 'search',
+        subject: 'test search',
+        context,
+      });
+
+      const [command, data] = ReactGA.testModeAPI.calls[1];
```

Comment:
Maybe good to check that only one call was made in this case?

Reply:
Added